src/app-layout/visual-refresh-toolbar/toolbar/drawer-triggers.tsx

```diff
@@ -112,15 +112,18 @@ export function DrawerTriggers({
               ariaLabel={splitPanelToggleProps.ariaLabel}
               ariaControls={splitPanelToggleProps.controlId}
               ariaExpanded={splitPanelToggleProps.active}
-              className={clsx(styles['drawers-trigger'], splitPanelTestUtilStyles['open-button'])}
+              className={clsx(
+                styles['drawers-trigger'],
+                testutilStyles['drawers-trigger'],
+                splitPanelTestUtilStyles['open-button']
+              )}
               iconName={splitPanelToggleProps.position === 'side' ? 'view-vertical' : 'view-horizontal'}
               onClick={() => onSplitPanelToggle?.()}
               selected={splitPanelToggleProps.active}
               ref={splitPanelFocusRef}
               hasTooltip={true}
               testId={`awsui-app-layout-trigger-slide-panel`}
               isMobile={isMobile}
-              isForPreviousActiveDrawer={splitPanelToggleProps.active}
```

Comment:
Why is this removed?

Reply:
because this is for the split panel and has its own consistent refs vs the drawers that share one toggleRef. The focus handler in the button has a special condition for the split panel